### PR TITLE
update cmdline tools and platform api

### PR DESCRIPTION
**Pull request by Jiyvn**
- update cmdline-tools to version 7583922_latest
- update platform api to 29

### CODE REVIEW

File: Appium/Dockerfile [hunk 1]
```diff
@@ -65,15 +65,17 @@ ENV JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64/jre" \
...
+ARG ANDROID_PLATFORM_VERSION="android-29"
 ENV ANDROID_HOME=/root
 
 RUN wget -O tools.zip https://dl.google.com/android/repository/${SDK_VERSION}.zip && \
     unzip tools.zip && rm tools.zip && \
     chmod a+x -R $ANDROID_HOME && \
     chown -R root:root $ANDROID_HOME
 
-ENV PATH=$PATH:$ANDROID_HOME/tools:$ANDROID_HOME/tools/bin
+RUN mv cmdline-tools tools && mkdir cmdline-tools && mv tools cmdline-tools/tools
```
Comment (mykola-mokhnach) [inline Appium/Dockerfile:76]: you could use `mkdir -p` to create directories recursively

File: Appium/Dockerfile [hunk 2]
```diff
@@ -98,6 +100,8 @@ RUN curl -sL https://deb.nodesource.com/setup_12.x | bash && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
     apt-get clean
 
+RUN appium driver install uiautomator2 && appium driver install xcuitest
```
Comment (mykola-mokhnach) [inline Appium/Dockerfile:103]: @jlipps can we install multiple drivers within single driver install` call?

Comment (jlipps) [reply Appium/Dockerfile:103]: no, but you can install multiple drivers at the same time as installing appium, i think it's `npm install -g appium@next --drivers=uiautomator2,xcuitest`

Comment (jlipps) [reply Appium/Dockerfile:103]: sadly, https://github.com/appium/appium/issues/16924

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by Jiyvn** [comment]
https://github.com/appium/appium/issues/16941

**Comment by js361014** [comment]
Please replace 
`CMD="xvfb-run appium --log $APPIUM_LOG"`
with
`CMD="xvfb-run appium server --base-path /wd/hub --log $APPIUM_LOG"`
in `entry_point.sh`

and `APPIUM_VERSION=next` with `APPIUM_VERSION=2.0.0-beta.42` in `Dockerfile`

**Comment by budtmo** [comment]
I will close the issue as there is no further response from the commiter and I provide Appium 2.0-beta for next release

**Pull request closed without merging**